authors/apps/articles/views.py

```diff
@@ -140,7 +152,8 @@ def update(self, request, *args, **kwargs):
                 [str(tag) for tag in article.tag_list.all()]
             ),
             "image_url": request.data.get('image_url', article.image_url),
-            "audio_url": request.data.get('audio_url', article.audio_url)
+            "audio_url": request.data.get('audio_url', article.audio_url),
+            "read_time": request.data.get('read_time', article.read_time)
```

Comment:
What happens when a user updates an article. Do you recalculate the read time?

Reply:
Thanx for pointing this out. The read time has to be recalculated upon saving the article changes. updates on the same have been made.